src/server/pps/server/api_server.go

```diff
@@ -1510,14 +1510,17 @@ func (a *apiServer) getLogsFromStats(pachClient *client.APIClient, request *pps.
 }
 
 func (a *apiServer) validatePipeline(pachClient *client.APIClient, pipelineInfo *pps.PipelineInfo) error {
-	if pipelineInfo.Pipeline == nil {
+	if pipelineInfo.Pipeline == nil || len(pipelineInfo.Pipeline.Name) < 1 {
```

Comment:
I would do this as `pipelineInfo.Pipeline.Name == ""`, not a big deal but I find that I tiny bit clearer.